bot/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # see https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
 ARG NODE_VERSION=node:16.9.1
 
-FROM segment/chamber:2.10.7 AS chamber
+# FROM segment/chamber:2.10.7 AS chamber
```

Comment:
i removed `chamber` from the dockerfile as its not needed anymore and the needed Secrets are being pulled from the [argocd-value-files](https://github.com/sidestream-tech/k8s-projects/blob/d9392f6c1be2b9381e745f6e5eef2f42f9786ccc/auctions-ui/staging/values-bot.yml#L62).